Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift

```diff
@@ -26,34 +51,29 @@ public final class FormToggleItemView: FormValueItemView<Bool, FormToggleItemSty
         observe(item.publisher) { [weak self] value in
             self?.switchControl.isOn = value
         }
-        
+
+        addSubviews()
+    }
+
+    // MARK: - Private
+
+    private func addSubviews() {
         addSubview(stackView)
-        stackView.adyen.anchor(inside: self.layoutMarginsGuide)
+        [titleLabel, switchControl].forEach(stackView.addArrangedSubview)
+
+        setupLayout()
     }
-    
-    /// :nodoc:
-    override public func reset() {
-        item.value = false
+
+    private func setupLayout() {
+        stackView.adyen.anchor(inside: layoutMarginsGuide)
     }
-    
-    // MARK: - Switch Control
-    
-    internal lazy var switchControl: UISwitch = {
-        let switchControl = UISwitch()
-        switchControl.isOn = item.value
-        switchControl.onTintColor = item.style.tintColor
-        switchControl.isAccessibilityElement = false
-        switchControl.addTarget(self, action: #selector(switchControlValueChanged), for: .valueChanged)
-        switchControl.setContentHuggingPriority(.required, for: .horizontal)
```

Comment:
shouldn't you keep `switchControl.setContentHuggingPriority` as well?

Reply:
You're right, it makes sense to keep the content hugging priority

Reply:
Added back